### DISCUSSION TITLE
how to add country to output file

### DISCUSSION THREAD

**Discussion by SOMEGUY333422** [iw4p/proxy-scraper#18] [Q&A]
how would you add the country of the proxy in the output text file?

**Comment by iw4p** [comment]
Do you mean to get the proxies of a specific country?

**Reply by SOMEGUY333422** [reply]
no so with the proxies it gives in the output.txt file how can i get it to also display the country

**Reply by iw4p** [reply]
If I add this feature, output file will lose the IP:PORT format.
